tests/test_processing_stack.py

```diff
@@ -165,3 +167,16 @@ def should_successfully_run_dataset_version_creation_process(
                 time.sleep(5)
 
             assert copy_job["Job"]["Status"] == "Complete", copy_job
+
+            with subtests.test(msg="Import Status Endpoint"):
+                status_resp = lambda_client.invoke(
+                    FunctionName=ResourceName.IMPORT_STATUS_ENDPOINT_FUNCTION_NAME.value,
+                    Payload=json.dumps(
+                        {"httpMethod": "GET", "body": {"execution_arn": execution["executionArn"]}}
+                    ).encode(),
+                    InvocationType="RequestResponse",
+                )
+                status_json_resp = json.load(status_resp["Payload"])
+                assert status_json_resp["statusCode"] == 200, status_json_resp
+                assert status_json_resp["body"]["validation_status"] == "SUCCEEDED"
+                assert status_json_resp["body"]["upload_status"] == "Complete"
```

Comment:
Each of these assertions should be in separate subtests, *or* the test should do a single assert on the whole response.